hooks/post-commit.py

```diff
@@ -5,7 +5,7 @@
 
 
 def run_command(command):
-    return subprocess.check_output(command.split(), shell=True).rstrip()
+    return subprocess.check_output(command.split()).rstrip()
```

Comment:
I don't remember why this was here (I think it was to do with aliasing and x-platform) but why did you take it out -- does it break something?